targets/xorg

```diff
@@ -197,6 +226,19 @@ if [ -z "$inteldriver" -a -n "$backport" ] && release -eq precise; then
         x11-session-utils x11-utils x11-xfs-utils x11-xkb-utils \
         x11-xserver-utils xauth xinit xfonts-utils xkb-data xorg-docs-core \
         xterm x11-common xinput
+elif [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    # Add use expand flags for the drivers we're installing
+    if echo "$installvideodrivers" | grep -q "$inteldriver"; then
```

Comment:
`if [ "${installvideodrivers#*"$inteldriver"}" != "$installvideodrivers" ]; then` should do the same thing